encoders/nlp/TransformerTorchEncoder/tests/test_transformertorchencoder.py

```diff
@@ -68,6 +68,19 @@ def test_encoding_results(test_metas, model_name, pooling_strategy, layer_index)
     else:
         assert np.allclose(encoded_data[0], encoded_data[1], atol=1e-5, rtol=1e-4)
 
+@pytest.mark.parametrize('acceleration', ["amp", "quant"])
```

Comment:
Hey @tadejsv,

Thank you very much again for your awesome contributions, can u please keep it single-quoted?